src/gui/WndMain.cpp

```diff
@@ -65,9 +65,36 @@
 
 static int gameLogoWidth, gameLogoHeight;
 static int splashLogoWidth, splashLogoHeight;
+
 static HANDLE hPersistedMemory = NULL;
 static LPVOID PersistedMemoryAddr = nullptr;
 
+void MapPersistedMemory()
+{
+	assert((hPersistedMemory == NULL) == (PersistedMemoryAddr == NULL));
+

```

Comment:
PersistedMemoryAddr should be checked against nullptr instead since it's a pointer. hPersistedMemory is an handle so that's fine with NULL.

Reply:
You're right, that should be corrected